web/components/PriorityClassList.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-row v-if="priorityclasses.length !== 0" no-gutters>
+    <v-col>
+      <v-card class="ma-2" outlined>
+        <v-card-title class="mb-1"> PriorityClasses </v-card-title>
+        <v-card-actions>
+          <v-chip
+            v-for="(p, i) in priorityclasses"
+            :key="i"
+            class="ma-2"
+            color="primary"
+            outlined
+            large
+            label
+            @click.stop="onClick(p)"
+          >
+            <img src="/sc.svg" height="40" alt="p.metadata.name" class="mr-2" />
```

Comment:
`sc.svg` is the image for strage classes.
I googled the icon for priority class, but it seems there is no icon for priority class. 😓
So, let's delete this `<img ... />`

Reply:
I have just deleted it